js/main.js

```diff
@@ -0,0 +1,42 @@
+var user="John Doe";
+console.log(user);
+var student="Olga";
+console.log(student);
+var user="student";
+console.log(user);
+let test=1;
+console.log(test);
+++test;
+console.log(test);
+console.log(test+1);
+console.log(test);
+--test;
+console.log(test);
+if (test)
+{
```

Comment:
I would recommend sometimes to add empty lines for easier reading of the code. For example between tasks or logic blocks